advanced_source/cpp_custom_ops.rst

```diff
@@ -255,13 +244,41 @@ first load the C++ library that holds the custom operator definition
 and then call the ``torch.library`` registration APIs. This can happen in one
 of two ways:
 
-1. If you're following this tutorial, importing the Python C extension module
-   we created will load the C++ custom operator definitions.
-2. If your C++ custom operator is located in a shared library object, you can
-   also use ``torch.ops.load_library("/path/to/library.so")`` to load it. This
-   is the blessed path for Python agnosticism, as you will not have a Python C
-   extension module to import. See `torchao __init__.py <https://github.com/pytorch/ao/blob/881e84b4398eddcea6fee4d911fc329a38b5cd69/torchao/__init__.py#L26-L28>`_
-   for an example.
+
+1. In this tutorial, our C++ custom operator is located in a shared library object,
+   and we use ``torch.ops.load_library("/path/to/library.so")`` to load it. This
+   is the blessed path for Python agnosticism, and you will not have a Python C
+   extension module to import. See our `extension_cpp/__init__.py <https://github.com/pytorch/extension-cpp/blob/e4c4eb822889ea67f191071fa627d750e04bf047/extension_cpp/__init__.py>`_
+   for an example:
+
+.. code-block:: python
+
+  import torch
+  from pathlib import Path
+
+  so_files = list(Path(__file__).parent.glob("_C*.so"))
+  assert (
+      len(so_files) == 1
+  ), f"Expected one _C*.so file, found {len(so_files)}"
+  torch.ops.load_library(so_files[0])
+
+  from . import ops
+
+
+2. You may also see other custom extensions importing the Python C extension module.
+   The module would be created in C++ and then imported in Python, like the code below.
+   This code is not guaranteed to use the stable limited CPython API and would block
+   your extension from building a Python-agnostic wheel! AVOID the following:
```

Comment:
AVOID feels like you are screaming at me =)
```suggestion
   your extension from building a Python-agnostic wheel! Please avoid the following:
```

Reply:
hehe i am ;) but i've rephrased